infrastructure-playbooks/purge-cluster.yml

```diff
@@ -478,6 +483,7 @@
       name: ceph-mon@{{ ansible_hostname }}
       state: stopped
       enabled: no
+    failed_when: false
```

Comment:
is it for idempotency purpose?

Reply:
kinda, I had to add these lines mostly because it failed before (the zsh expansion failed) and it did not purge everything. Since it failed half-way, it failed to purge anything in the second round because it could not find the systemd services.

btw: if you look a couple lines down, you'll notice we were already doing this for upstart.